src/Controls/src/Core/Border/Border.cs

```diff
@@ -64,10 +64,13 @@ void NotifyStrokeShapeChanges()
 
```

Comment:
Should the code that calls stop notify have this if statement?

https://github.com/dotnet/maui/blame/e3a38e231c5412831c8a968948fe0f92a1115a32/src/Controls/src/Core/Border/Border.cs#L51-L58

It seems like it should just always call stop notifying no matter what.

Reply:
I think both properties have a slight issue:
```
--if (newvalue is not null)
++if (oldvalue is not null)
	(bindable as Border)?.StopNotifyingStrokeShapeChanges();
```
I guess they were always this way, so I pushed this change.